src/api/app/controllers/request_controller.rb

```diff
@@ -70,6 +70,7 @@ def create
       @req.set_add_revision       if params[:addrevision].present?
       @req.set_ignore_delegate    if params[:ignore_delegate].present?
       @req.set_ignore_build_state if params[:ignore_build_state].present?
+      @req.enforce_branching      if params[:enforce_branching].present?
```

Comment:
```suggestion
      @req.enforce_branching = true      if params[:enforce_branching].present?
```
Then on `BsRequest` I would use an attribute writter, the rails way of writing setters (`attr_writer :enforce_branching`).

Reply:
I'd like to stay in sync with the code above, but feel free to change it.

Reply:
Sorry, stay what in sync?